packages/web/src/components/comments/CommentHeader.tsx

```diff
@@ -1,15 +1,75 @@
-import { Text } from '@audius/harmony'
+import { useContext, useState } from 'react'
+
+import { CommentSectionContext } from '@audius/common/context'
+import {
+  Flex,
+  IconButton,
+  IconKebabHorizontal,
+  PopupMenu,
+  PopupMenuItem,
+  Text
+} from '@audius/harmony'
+import { css, useTheme } from '@emotion/react'
+
+const messages = {
+  turnOffNotifs: 'Turn off notifications'
+}
+
+type CommentHeaderProps = {
+  commentCount?: number
+  isLoading?: boolean
+}
 
 export const CommentHeader = ({
   commentCount,
   isLoading
-}: {
-  commentCount?: number
-  isLoading?: boolean
-}) => {
+}: CommentHeaderProps) => {
+  const { handleMuteEntityNotifications, isEntityOwner } = useContext(
+    CommentSectionContext
+  )!
+  const { motion } = useTheme()
+  const popupMenuItems: PopupMenuItem[] = [
+    { onClick: handleMuteEntityNotifications, text: messages.turnOffNotifs }
+  ]
+  const [isPopupOpen, setIsPopupOpen] = useState(false)
+
   return (
-    <Text variant='title' size='l'>
-      Comments ({!isLoading ? commentCount : '...'})
-    </Text>
+    <Flex
+      justifyContent='space-between'
+      w='100%'
+      css={css`
+        &:hover .kebabIcon {
+          opacity: 1;
+        }
+      `}
+    >
+      <Text variant='title' size='l'>
+        Comments ({!isLoading ? commentCount : '...'})
+      </Text>
+      {isEntityOwner && !isLoading ? (
+        <PopupMenu
+          items={popupMenuItems}
+          onClose={() => setIsPopupOpen(false)}
+          renderTrigger={(anchorRef, triggerPopup) => (
+            <IconButton
+              aria-label='Show comment options'
+              icon={IconKebabHorizontal}
+              color='subdued'
+              ref={anchorRef}
+              css={{
+                cursor: 'pointer',
```

Comment:
Weird that IconButton doesn't have `pointer` as default

Reply:
yea i cant think of a reason that any button wouldn't have pointer as the default